tmt/steps/execute/__init__.py

```diff
@@ -344,6 +344,7 @@ def handle_reboot(self) -> bool:
 
         reboot_command: Optional[ShellScript] = None
         timeout: Optional[int] = None
+        flush_artifacts: Optional[bool] = False
```

Comment:
`Optional` is not needed, the value ill always be a boolean - if it's not defined in the reboot data package, the default - `flush_artifacts` as set here - would be used, returning `False`.